cmd/frontend/graphqlbackend/external_service_collaborators.go

```diff
@@ -57,7 +58,8 @@ func (r *externalServiceResolver) InvitableCollaborators(ctx context.Context) ([
 	}
 	baseURL = extsvc.NormalizeBaseURL(baseURL)
 	githubUrl, _ := github.APIRoot(baseURL)
-	client := github.NewV4Client(githubUrl, &auth.OAuthBearerToken{Token: githubCfg.Token}, nil)
+	newCache := func(key string, ttl int) github.Cache { return rcache.NewWithTTL(key, ttl) }
```

Comment:
```suggestion
	newCache := func(key string, ttl int) github.Cache { 
	    return rcache.NewWithTTL(key, ttl) 
	}
```

This might be easier to read quickly.